[Docs] Clarify Zadig usage in FAQ Docs (#6360)
qmk/qmk_firmware: docs/faq_build.md

## Unknown Device for DFU Bootloader

> If you're using Windows to flash your keyboard, and you are running into issues, it is probably because the wrong drivers are installed for the bootloader.

Reviewer: ```suggestion
Issues encountered when flashing keyboards on Windows are most often due to having the wrong drivers installed for the bootloader. 
```

> Re-running the installation script for MSYS2 may help (eg run `util/qmk_install.sh` from MSYS2/WSL) or reinstalling the QMK Toolbox may fix the issue. Alternatively, you can download the [`qmk_driver_installer`](https://github.com/qmk/qmk_driver_installer) package and run that.

Reviewer: ```suggestion
Re-running the installation script for MSYS2 may help (eg run `util/qmk_install.sh` from MSYS2/WSL) or reinstalling the QMK Toolbox may fix the issue. Alternatively, you can download and run the [`qmk_driver_installer`](https://github.com/qmk/qmk_driver_installer) package.
```

> If that doesn't work, then you may need to grab the [Zadig Utility](https://zadig.akeo.ie/). Download this, and run it on the system.  Then, you will need to reset your board into bootloader mode.  After that, find the device in question, if the device doesn't show up in the list (or nothing shows up in the list), then you may need to enable the `List all devices` option in the `Options` menu.

Reviewer: ```suggestion
If that doesn't work, then you may need to grab the [Zadig Utility](https://zadig.akeo.ie/). Download this, and run it on the system.  Then, you will need to reset your board into bootloader mode.  After that, locate the device in question. If the device doesn't show up in the list (or nothing shows up in the list), you may need to enable the `List all devices` option in the `Options` menu.
```

> From here, you will need to know what type of controller that your board is using.  You may see it listed in the Device Manager as `ATmega32U4` device (which is an AVR board), or an `STM32` device (Which is an ARM board).  For AVR boards, you will want the `libusb-win32` for the driver, and for ARM boards, you will want the `WinUSB` driver.  Once you have the right driver type selected, click on the `Replace Driver` button, and then unplug your board, plug it back in, and reset it again.

Reviewer: ```suggestion
From here, you will need to know what type of controller the board is using.  You may see it listed in the Device Manager as `ATmega32U4` device (which is an AVR board), or an `STM32` device (Which is an ARM board).  For AVR boards, use `libusb-win32` for the driver. For ARM boards, use the `WinUSB` driver.  Once the correct driver type has been selected, click on the `Replace Driver` button, unplug your board, plug it back in, and reset it again.
```
